src/sys/mod.rs

```diff
@@ -1,3 +1,6 @@
+#[cfg(all(target_os = "linux",
+          any(target_arch = "x86", target_arch = "x86_64")))]
```

Comment:
They could be added. However, testing a struct if you do not have a platform involves a full CI run, so it is a bit tedious. I will add a comment for now.